src/components/elements/Inputs/FileInput/FileInput.tsx

```diff
@@ -117,30 +125,54 @@ const FileInput = (props: FileInputProps) => {
             multiple={props.allowMultiple}
             accept={accept}
           />
-          <div className="m-auto flex w-fit items-center justify-center gap-3">
-            <FileCardContent
-              title={labelText}
-              subtitle={`${t("Drag and drop or")} <span className="text-primary underline">${t(
-                "browse your device"
-              )}</span>`}
-              thumbnailClassName="fill-primary"
-              thumbnailContainerClassName="bg-primary-100"
-            />
-          </div>
+          <When condition={!variant.snapshotPanel}>
+            <div className="m-auto flex w-fit items-center justify-center gap-3">
+              <FileCardContent
+                title={labelText}
+                subtitle={`${t("Drag and drop or")} <span className="text-primary underline">${t(
+                  "browse your device"
+                )}</span>`}
+                thumbnailClassName="fill-primary"
+                thumbnailContainerClassName="bg-primary-100"
+              />
+            </div>
+          </When>
+          <When condition={variant.snapshotPanel}>
+            <Icon name={IconNames.UPLOAD_CLOUD} className="mb-4 h-5 w-5" />
+            <div className="flex flex-col">
+              <Text variant="text-12-bold" className="text-center text-primary">
+                Click to upload
+              </Text>
+              <Text variant="text-12-light" className="text-center">
+                or
+              </Text>
+              <Text variant="text-12-light" className="max-w-[210px] text-center">
+                {props.descriptionInput}
```

Comment:
This set of text needs translation and also has a translation problem in that it's making an assumption about sentence order similar to the comment above. I'm not sure what the solution is without seeing the designs... Hopefully something can be composed similar to the example above, with spans to note differences in text styles.

Reply:
is not working in this case
![image](https://github.com/wri/wri-terramatch-website/assets/139272044/610fa331-103e-4846-8274-c2495a9ed532)
i just add the t
here is the design:
![image](https://github.com/wri/wri-terramatch-website/assets/139272044/082580c8-b411-4f56-9d2b-c449c8625c58)